index.html

```diff
@@ -6,11 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
-    </head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
 <body>
-    <header>
-        <img src="" alt="">
+    <header id="toolbar">
+        <img src="images/ic_menu.svg" alt="Menu">
+        <a href=""></a> 
     </header>
-    <h1>Felipe Haybar</h1>
+        <h1>Felipe Haybar</h1>
+    <section id="headline">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+        <br>If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    </section>
+    <button class="blackBoton">
+        <a class="textBoton" href="#">Start collaboration</a>
+    </button>
```

Comment:
- All this block of code should be a section not just lines 19-22. It represents a generic standalone section of the document so it should be inside a `section` tag. Kindly modify this. :nerd_face: :nerd_face: 
- Please ensure that you are using the same margins provided in the Figma template. The right margin is bigger than the left margin. Kindly reduce it. :eyes: :eyes: 

Your design
![image](https://user-images.githubusercontent.com/98363075/199642376-3fb28a7e-25d5-4913-a43d-2f4682814617.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/199642458-f256c20a-c74c-4c50-9d6b-7851fba18e52.png)

Reply:
I´m resolving this been my project responsive